### ISSUE TITLE
Add data resources for all resources

### ISSUE THREAD

**Issue by scottford-io** [mondoohq/terraform-provider-mondoo#45]
**Is your feature request related to a problem? Please describe.**
We need data resources to use resources provisioned by the Mondoo Terraform Provider. Data resources allow us to reference dynamically generated resources/configurations.

**Describe the solution you'd like**

- [ ] Service Account (json) - This needs to be marked as `sensitive` 
- [ ] Space ID

**Describe alternatives you've considered**

**Additional context**

**Comment by chris-rock** [comment]
@scottford-io Can you provide an example where you need this? You can get the data from the generated resources without data resource. So I'd like to understand the use case better.

**Comment by ehaselwanter** [comment]
I guess this is already answered? If not read `data sources` as a least privilege access to the information behind the provider encapsulated information aka `read access`. One example is that you create the tf resources in one workspace but want to access the information in another workspace with different RBAC .... and it does not need to be tied to a/the resource abstraction

**Comment by chris-rock** [comment]
@ehaselwanter I agree that we need data resources. I tried to find out, which data resources we actually need and which one are helpful for users.
